server/src/user-org/user-org.controller.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  HttpException,
+  HttpStatus,
+  UseGuards,
+} from '@nestjs/common';
+import { DeleteResult } from 'typeorm';
+
+import { JwtAuthGuard } from '../auth/guards/jwt-auth.guard';
+import type { CreateUserOrganizationDto } from './dto/create-user-org.dto';
+import type { UpdateUserOrganizationDto } from './dto/update-user-org.dto';
+import { UserOrganization } from './entities/user-org.entitiy';
+import { UserOrganizationsService } from './user-org.service';
+
+@Controller('userOrganizations')
+export class UserOrganizationsController {
+  constructor(private readonly userOrganizationsService: UserOrganizationsService) {}
+  @Post()
+  async create(
+    @Body() createUserOrganizationsDto: CreateUserOrganizationDto,
+  ): Promise<UserOrganization> {
+    try {
+      const user_org = await this.userOrganizationsService.create(createUserOrganizationsDto);
+      return user_org;
+    } catch (error) {
+      throw new HttpException(
+        { status: HttpStatus.CONFLICT, message: `${error}` },
+        HttpStatus.CONFLICT,
+      );
+    }
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Get()
+  async findAll() {
+    const allUserOrgs = await this.userOrganizationsService.findAll();
+    return allUserOrgs;
+  }
+
+  @UseGuards(JwtAuthGuard)
```

Comment:
we no longer have a `JwtAuthGuard` use `CookieGuard` from https://github.com/Nonprofit-Exchange-Hub/web-app/pull/91